include/sot/core/parameter-server.hh

```diff
@@ -99,9 +100,36 @@ public:
   /// @}
   /// \name Commands related to the model
   /// @{
+  template <typename Type>
   void setParameter(const std::string &ParameterName,
-                    const std::string &ParameterValue);
-  std::string getParameter(const std::string &ParameterName);
+                    const Type &ParameterValue)
+  {
+    if (!m_initSucceeded) {
+      std::ostringstream oss;
+      oss << ParameterValue;
+      SEND_WARNING_STREAM_MSG("Cannot set parameter " + ParameterName + " to " +
+                              oss.str().c_str() +
+                              " before initialization!");
```

Comment:
What about the following one-liner ?
```suggestion
      DYNAMIC_GRAPH_ENTITY_WARNING(*this) << "Cannot set parameter " << ParameterName
        << " to " << ParameterValue << " before initialization!";
```

Reply:
Note that I removed the stream. I don't think it makes sense here.

Reply:
I cannot use the DYNAMIC_GRAPH_ENTITY_WARNING because the class RobotUtils is not an entity.

Reply:
The class here is ParameterServer, which is an entity.

Reply:
You are right. Applied in commit f7f8ebb